staging/src/k8s.io/apiserver/pkg/server/serve.go

```diff
@@ -114,7 +115,9 @@ func RunServer(server *http.Server, network string, stopCh <-chan struct{}) (int
 	// Stop the server by closing the listener
 	go func() {
 		<-stopCh
-		ln.Close()
+		ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
```

Comment:
I assume these 60 seconds come from the WithTimeout 60 seconds? I would like to see those two values linked.

Reply:
Yes, they are same meaning . Should be linked.